src/themes.c

```
@@ -1009,6 +1009,13 @@ void thmTriggerCoverflowAnim(int dir)
     cfAnimStartTime = clock();
 }
 
+// Exposes the slide state so per-frame diagnostic overlays (gui.c guiDrawOverlays) can step
+// aside mid-slide; see the gate there for the VRAM/TexManager rationale.
+int thmCoverflowIsAnimating(void)
```

Cppcheck (2.21.0)

[style] The function 'thmCoverflowIsAnimating' is never used.

(unusedFunction)